### DISCUSSION TITLE
How to copy a virtual machine image?

### DISCUSSION THREAD

**Discussion by why168** [lima-vm/lima#2550] [Q&A]
I have 4 images locally, all of which have installed a lot of software. I don't want to reinstall another image to set up the environment from 0 to 1. I want to copy and use the existing images directly. 

```shell
(base)  ~/ limactl ls
NAME                      STATUS     SSH            VMTYPE    ARCH       CPUS    MEMORY    DISK      DIR
default                   Stopped    127.0.0.1:0    qemu      aarch64    4       4GiB      100GiB    ~/.lima/default
docker-rootful-local      Stopped    127.0.0.1:0    vz        aarch64    8       16GiB     60GiB     ~/.lima/docker-rootful-local
docker-rootful-rosetta    Stopped    127.0.0.1:0    vz        aarch64    8       16GiB     60GiB     ~/.lima/docker-rootful-rosetta
ubuntu-gateway-inlet      Stopped    127.0.0.1:0    vz        aarch64    1       1GiB      10GiB     ~/.lima/ubuntu-gateway-inlet
```

**Comment by why168** [comment]
@AkihiroSuda  Excuse me. I need help

**Reply by AkihiroSuda** [reply]
You may try to just copy the `~/.lima/<INSTANCE>` dir manually to see if it works, but it is currently out of support.

**Reply by jandubois** [reply]
Also make sure the instance is stopped before copying the directories. Otherwise the copies of the disk images may be corrupted.
